database-postgres/seeds/addData.js

```diff
@@ -0,0 +1,12 @@
+const {importData} = require('../importData.js');
+
+exports.seed = function(knex) {
+  // Deletes ALL existing entries
```

Comment:
I know you might not be done with this section yet, but be careful to delete the old rows from the table first, like the generated seed function has scaffolded out. Otherwise if running `knex seed:run` by itself, you're likely to encounter an error. (Note that you _probably_ won't encounter any errors if always running `knex migrate:rollback && knex migrate:latest && knex seed:run` together with the seed function you currently have)